### PR TITLE
Add convert currency and tests

### PR DESCRIPTION
**Pull request by Edge7481**
### [Pivotal Tracker Link][tracker]

[tracker]: https://www.pivotaltracker.com/story/show/187348127

## What this PR does:

This pull request implements
a static method in Currency_Conversion.rb that converts a Money object of a currency to a Money object of another currency, since as of right now there's no functionality to actually convert. It will be used to compute payments later
### Include screenshots, videos, etc.
N/A

### How should this PR be tested?

* Is there a deploy we can view?
* What do the specs/features test?

I added 2 tests that tests the path that there is a valid currency in the table, and the path the there is no currency in the table.

* Are there edge cases to watch out for?

#### Are there any complications to deploying this?
No

### Checklist:

- [x] Has this been deployed to a staging environment or reviewed by a customer?
- [x] Tag someone for code review (either a coach / team member)
- [x] I have renamed the branch to match PivotTracker's suggested one (necessary for BlueJay)

### CODE REVIEW

File: app/models/currency_conversion.rb [hunk 1]
```diff
@@ -25,4 +25,15 @@ class CurrencyConversion < ApplicationRecord
   belongs_to :conference
   validates :rate, numericality: { greater_than: 0 }
   validates :from_currency, uniqueness: { scope: :to_currency }, on: :create
+
+  def self.convert_currency(conference, amount, from_currency, to_currency)
+    conversion = conference.currency_conversions.find_by(from_currency: from_currency, to_currency: to_currency)
+    if conversion
+      Money.add_rate(from_currency, to_currency, conversion.rate)
+      amount.exchange_to(to_currency)
+    else
```
Comment (fp456) [inline app/models/currency_conversion.rb:34]: Maybe consider returning -1 instead instead of returning an amount without conversion. Whenever you use this method convert_currency, you can check for the amount not being -1

Comment (fp456) [reply app/models/currency_conversion.rb:34]: if you choose this path, remember to update the tests below

**Comment by Edge7481** [comment]
Updated the bad path to -1 now. Should work better for validation!
